src/preprocessing/passes/real_to_int.cpp

```diff
@@ -171,8 +171,11 @@ Node RealToInt::realToIntInternal(TNode n, NodeMap& cache, std::vector<Node>& va
                              nm->integerType(),
                              "Variable introduced in realToIntInternal pass");
           var_eq.push_back(n.eqNode(ret));
-          TheoryModel* m = d_preprocContext->getTheoryEngine()->getModel();
```

Comment:
Since this line touched models, I suggest adding a `--produce-models` and `(get-model)` to the test. Is that possible?